incubator/oauth2-proxy/templates/configmap.yaml

```diff
@@ -43,6 +43,8 @@ data:
       ## when disabled the upstream Host is used as the Host Header
       pass_host_header = {{ .Values.app.passHostHeader }}
 
+      set_xauthrequest = {{ .Values.app.setXAuthRequest }}
```

Comment:
should we rename it to convey that it's related to headers?

```
set_xauthrequestheaders = {{ .Values.app.setXAuthRequestHeaders }}
```

Reply:
I prefer to just map configs that we have for oauth 2 proxy.